gensim/models/wrappers/ldamallet.py

```diff
@@ -173,13 +156,14 @@ def __getitem__(self, bow, iterations=100):
         cmd = cmd % (self.fcorpusmallet() + '.infer', self.finferencer(), self.fdoctopics() + '.infer', iterations)
         logger.info("inferring topics with MALLET LDA '%s'" % cmd)
         check_output(cmd, shell=True)
-        result = list(read_doctopics(self.fdoctopics() + '.infer'))
+        result = list(self.read_doctopics(self.fdoctopics() + '.infer'))
         return result if is_corpus else result[0]
 
 
     def load_word_topics(self):
         logger.info("loading assigned topics from %s" % self.fstate())
         wordtopics = numpy.zeros((self.num_topics, self.num_terms), dtype=numpy.float32)
+        word2id = dict((v, k) for k, v in iteritems(self.id2word))
```

Comment:
wondering why the check for token2id attr was dropped (L191 previously)? seems like it would eliminate the need for a new duplicate one when possible. could be something like:

```
if hasattr(self.id2word, 'token2id'):
    word2id = self.id2word.token2id
else:
    word2id = # current line
```